docs/docs/forms.mdx

```diff
@@ -320,6 +320,19 @@ The following example shows the implementation of a form which extracts a slot
 The method `extract_outdoor_seating` sets the slot `outdoor_seating` based on whether
 the keyword `outdoor` was present in the last user message.
 
+:::note
+Always return a value from the `extract_` methods. If you didn't found a match or you can't
+be sure that there is a new value, you can return the previous value using:
+
+```python
+return {"outdoor_seating": tracker.slots.get("outdoor_seating")}
```

Comment:
shouldn't this be `None` directly if you can't set the slot, rather than set the slot to the existing value?